src/puppeteer/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM node:22-bullseye-slim
```

Comment:
any particular reason we use bullseye here and bookworm for uv? Also just in general, would it be more ergonomic to use alpine? Not sure if that would work. I assume moving to alpine would cause more issues due to musl and static linking.

Reply:
@ColinMcNeil did you go with bullseye because of chromium?
I know we went with debian over alpine because of the chromium dependency. In the long run, I'd like to discuss a separate container for container but it almost feels like we'd need to explore that as an option first in puppeteer so I think we should leave that out for now.
Colin will know whether bookworm could be an option.

Reply:
Tested with bookworm and it worked fine for the puppeteer MCP